auslib/db.py

```diff
@@ -461,6 +461,7 @@ def __init__(self, dialect, metadata, baseTable):
         self.table = Table('%s_history' % baseTable.t.name, metadata,
                            Column('change_id', Integer, primary_key=True, autoincrement=True),
                            Column('changed_by', String(100), nullable=False),
+                           mysql_row_format='COMPRESSED',
```

Comment:
Does this code still work if the db is not mysql? We don't care about that as much as we did in the past, but it would be good to not knowingly bust support for other dbs.